src/dsp_tools/cli.py

```diff
@@ -123,6 +124,7 @@ def _make_parser(
     )
     parser_upload.add_argument("-v", "--verbose", action="store_true", help=verbose_text)
     parser_upload.add_argument("-m", "--metrics", action="store_true", help="write metrics into a 'metrics' folder")
+    parser_upload.add_argument("-d", "--dump", action="store_true", help="write every request to DSP-API into a file")
```

Comment:
I would assume it dumps requests to SIPI too, so the description is technically not entirely correct. (though maybe from a pragmatic point of view clear enough)